### ISSUE TITLE
Take the pilot survey and give us feedback

### ISSUE THREAD

**Issue by dasaderi** [dasaderi/WhyNotOpenScience#7]
We will have a pilot survey ready for the #mozsprint (June 1-2, 2017). We would really appreciate it if you could take it and give us feedback on it. Thank you!

**Comment by chartgerink** [comment]
For those wanting to participate, here is the link (adding for ease-of-access)
https://docs.google.com/forms/d/e/1FAIpQLSfdGDWGYsK5hGgIPT3jbM7OheOf9iwZrwe74cokdEAZim60gA/viewform?c=0&w=1

# From the contribution guidelines
If you want to take the pilot survey and provide us with feedback on it then please:

- Raise an issue. Raise an issue called Taking pilot survey.
- Access the pilot survey. Fill in the survey and feedback sections (if you wish).
- Comment on your issue with Survey done!
- Comment on your issue with any specific comments/questions or feedback. Anything you want to tell us will help! By commenting on the issue, we can start discussing.

**Comment by chartgerink** [comment]
I was working on issue #11 but my computer froze, so there will be two responses from me in the test dataset (just FYI)

**Comment by amiefairs** [comment]
Not a problem! :)

**Comment by jvallera** [comment]
@ottagit @edovio @dunebuggie @vyaspranjal33 @LappleApple @poush

This survey could use your feedback - it is investigating how/why open science may be a challenging topic for people to understand or agree with. If you have time today or tomorrow it woule be great if you could fill out the survey then create an issue with feedback on how they might be able to improve the survey linked above. more info in the contribute file.

**Comment by amiefairs** [comment]
Hi @jvallera  and all others mentioned - just a big super huge thank you in advance! If you have time to spare today or tomorrow to contribute that would be wonderful, but if not we would still love to hear from you whenever you have the time.

**Comment by dasaderi** [comment]
Yes, please more feedback! :) Thank you @jvallera for advertising it.

**Comment by dasaderi** [comment]
@chartgerink Thank you for putting the link and reiterate the contribution guidelines.
